### PR TITLE
Featrure/online bestellung

### PR DESCRIPTION
**Pull request by GeBon22**
Please dont mind the styling issues as far as they occure. 
Next step is a code refactoring in which I wild import/export the basic components, so I have a cleaner code per page and can simplify things.

### CODE REVIEW

File: pages/bestellung/ruecktritt.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  width: 40px;
+  height: 50px;
+  position: absolute;
+  top: 7.5rem;
+  right: 2rem;
+`;
+
+const StyledZurueckButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/ruecktritt.js]: Don't use px for font sizes.

File: pages/bestellung/ruecktritt.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 5vh;
+  width: 30vw;
+  cursor: pointer;
+  margin-bottom: 20px;
+`;
+
+const StyledKopierenButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/ruecktritt.js]: Don't use px for font sizes.

File: pages/bestellung/ruecktritt.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  width: 40vw;
+  position: relative;
+  cursor: pointer;
+`;
+
+const StyledArticle = styled.article`
+  text-align: justify;
+  text-align-last: none;
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/ruecktritt.js]: Don't use px for font sizes.

File: pages/bestellung/schaden.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  height: 75vh;
+  margin-top: 2rem;
+  margin-bottom: 10px;
+  overflow: auto;
+`;
+
+const StyledArticle = styled.article`
+  text-align: center;
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/schaden.js]: Don't use px for font sizes.

File: pages/bestellung/schaden.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  font-weight: 300;
+  font-style: normal;
+  text-decoration: none;
+  padding: 10px;
+`;
+
+const StyledBreadcrumb = styled.h3`
+  color: #572887;
+  font-size: 15px;
```
Comment (stefankorder) [inline pages/bestellung/schaden.js]: Don't use px for font sizes.

File: pages/bestellung/schaden.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  left: 2rem;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledZurueckButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/schaden.js]: Don't use px for font sizes.

File: pages/bestellung/verzug.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  width: 90vw;
+  height: 75vh;
+  gap: 20px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+`;
+
+const StyledBreadcrumb = styled.h3`
+  color: #572887;
+  font-size: 15px;
```
Comment (stefankorder) [inline pages/bestellung/verzug.js]: Don't use px for font sizes.

File: pages/bestellung/verzug.js [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+  font-weight: 400;
+  position: absolute;
+  top: 8rem;
+  left: 2rem;
+`;
+
+const StyledArticle = styled.article`
+  text-align: center;
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/verzug.js]: Don't use px for font sizes.

File: pages/bestellung/verzug.js [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+  padding: 10px;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledZurueckButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/verzug.js]: Don't use px for font sizes.

File: pages/bestellung/ruecktritt.js [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+  width: 90vw;
+  height: 75vh;
+  margin-top: 1rem;
+  margin-bottom: 10px;
+  overflow: auto;
+`;
+
+const StyledBreadcrumb = styled.h3`
+  color: #572887;
+  font-size: 15px;
```
Comment (stefankorder) [inline pages/bestellung/ruecktritt.js]: Don't use px for font sizes.

File: pages/bestellung/irrtum.js [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+  left: 2rem;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledZurueckButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/irrtum.js]: Don't use px for font sizes.

File: pages/bestellung/irrtum.js [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+  font-weight: 300;
+  font-style: normal;
+  text-decoration: none;
+  padding: 10px;
+`;
+
+const StyledBreadcrumb = styled.h3`
+  color: #572887;
+  font-size: 15px;
```
Comment (stefankorder) [inline pages/bestellung/irrtum.js]: Don't use px for font sizes.

File: pages/bestellung/irrtum.js [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+  height: 75vh;
+  margin-top: 2rem;
+  margin-bottom: 10px;
+  overflow: auto;
+`;
+
+const StyledArticle = styled.article`
+  text-align: center;
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/irrtum.js]: Don't use px for font sizes.

File: pages/bestellung/index.js [hunk 14]
```diff
@@ -14,13 +34,58 @@ const StyledBody = styled.div`
...
+  cursor: pointer;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const StyledZurueckButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/index.js]: Don't use px for font sizes.

File: pages/bestellung/index.js [hunk 15]
```diff
@@ -14,13 +34,58 @@ const StyledBody = styled.div`
...
+  font-size: 15px;
+  font-weight: 400;
+  position: absolute;
+  top: 125px;
+  left: 37px;
+`;
+
+const StyledButton = styled.button`
+  color: #572887;
+  font-size: 18px;
```
Comment (stefankorder) [inline pages/bestellung/index.js]: Don't use px for font sizes.

File: pages/bestellung/index.js [hunk 16]
```diff
@@ -14,13 +34,58 @@ const StyledBody = styled.div`
...
   width: 90vw;
   height: 75vh;
-  gap: 20px;
   margin-top: 10px;
   margin-bottom: 10px;
 `;
+
+const StyledBreadcrumb = styled.h3`
+  color: #572887;
+  font-size: 15px;
```
Comment (stefankorder) [inline pages/bestellung/index.js]: Don't use px for font sizes.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Try to be a little more structured with your styling units.

File: pages/bestellung/ruecktritt.js [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+  background-color: #e6ceee;
+  border-radius: 10px;
+  border: 1px solid rgba(163, 134, 182, 0.5);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 15vh;
+  width: 40vw;
+  position: relative;
+  cursor: pointer;
+`;
+
```
Comment (actyralf) [inline pages/bestellung/ruecktritt.js:115]: Please consider to create a more generic `StyledButton` component that can be used in different scenarios.
The buttons differ only in width and height, you can configure these as props.

File: pages/bestellung/verzug.js [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+  cursor: pointer;
+`;
+
+const StyledIcon = styled(Icon)`
+  width: 40px;
+  height: 50px;
+  position: absolute;
+  top: 6.5rem;
+  right: 2rem;
+`;
```
Comment (actyralf) [inline pages/bestellung/verzug.js:91]: Your styled components seem to be copies of the versions in "schaden.js". Please consider moving the components in separate files and import them here.

**Review by actyralf — CHANGES_REQUESTED** [review]
Hey Georg,

please have a look at my comments - you can simplify your code a lot!

File: pages/bestellung/verzug.js (hunk 18 above)
Comment (GeBon22) [reply pages/bestellung/verzug.js:91]: yes, as I stated up top, this is the next step - I have made an extra US to simplify the code and components - thats probably why Stefan did not mentioned it above. Thank you :-)

**Pull request merged**